### PR TITLE
chore: better description for built-in negated flags

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**
chore

**Did you add tests for your changes?**
Nope

**If relevant, did you update the documentation?**
Already up to date.

**Summary**
A better description for built-in negated flags.

Screenshot 2020-12-04 at 6 12 38 PM

**Does this PR introduce a breaking change?**
Nope

**Other information**
N/A

### CODE REVIEW

File: packages/webpack-cli/lib/utils/cli-flags.js [hunk 1]
```diff
@@ -129,6 +129,7 @@ const builtInFlags = [
         type: Boolean,
         negative: true,
         description: 'Enables Hot Module Replacement',
```
Comment (alexander-akait) [inline packages/webpack-cli/lib/utils/cli-flags.js:131]: `Enables/Disables Hot Module Replacement`, we are using it for any boolean flags with negative in docs

Comment (jamesgeorge007) [reply packages/webpack-cli/lib/utils/cli-flags.js:131]: Currently, this is how the description shows up for built-in negated flags:-

Screenshot 2020-12-04 at 7 13 24 PM

Comment (alexander-akait) [reply packages/webpack-cli/lib/utils/cli-flags.js:131]: oh, yep, right

Comment (alexander-akait) [reply packages/webpack-cli/lib/utils/cli-flags.js:131]: Anyway we should solve this for all negative flag

Comment (jamesgeorge007) [reply packages/webpack-cli/lib/utils/cli-flags.js:131]: Currently, this isn't possible for core flags since we aren't receiving enough information.

Comment (alexander-akait) [reply packages/webpack-cli/lib/utils/cli-flags.js:131]: What is not enouth?

Comment (jamesgeorge007) [reply packages/webpack-cli/lib/utils/cli-flags.js:131]: We already have the built-in flags as part of [cli-flags](https://github.com/webpack/webpack-cli/blob/master/packages/webpack-cli/lib/utils/cli-flags.js), and hence adding a key corresponding to the description for negated flags would do. However, this isn't the case with core flags; the description for a negated flag is to be made available.

Comment (jamesgeorge007) [reply packages/webpack-cli/lib/utils/cli-flags.js:131]: We already have the built-in flags as part of [cli-flags](https://github.com/webpack/webpack-cli/blob/master/packages/webpack-cli/lib/utils/cli-flags.js), and hence adding a key corresponding to the description for negated flags would do. However, this isn't the case with core flags; the description for a negated flag is to be made available.

Comment (alexander-akait) [reply packages/webpack-cli/lib/utils/cli-flags.js:131]: What we output for `--no-amd`?

Comment (jamesgeorge007) [reply packages/webpack-cli/lib/utils/cli-flags.js:131]: Negates `amd`

Comment (alexander-akait) [reply packages/webpack-cli/lib/utils/cli-flags.js:131]: Can you provide example? For help output

Comment (jamesgeorge007) [reply packages/webpack-cli/lib/utils/cli-flags.js:131]: The description for built-in flags show up as provided, while core flags go by the format `Negates ${flag.name}`.

Screenshot 2020-12-04 at 6 12 38 PM

**Review by alexander-akait — DISMISSED** [review]

File: packages/webpack-cli/lib/utils/cli-flags.js [hunk 2]
```diff
@@ -182,6 +183,7 @@ const builtInFlags = [
         type: Boolean,
         negative: true,
         description: 'Enable/Disable colors on console',
```
Comment (alexander-akait) [inline packages/webpack-cli/lib/utils/cli-flags.js:185]: I think we should change it to `Enable`

Comment (jamesgeorge007) [reply packages/webpack-cli/lib/utils/cli-flags.js:185]: #2188

File: packages/webpack-cli/lib/groups/runHelp.js [hunk 3]
```diff
@@ -93,7 +93,9 @@ const outputHelp = (args) => {
         const negatedFlags = flags
             .filter((flag) => flag.negative)
             .reduce((allFlags, flag) => {
-                return [...allFlags, { name: `no-${flag.name}`, description: `Negates ${flag.name}`, type: Boolean }];
+                // Use available description for built-in negated flags
+                const description = flag.negatedDescription ? flag.negatedDescription : `Negates ${flag.name}`;
+                return [...allFlags, { name: `no-${flag.name}`, description, type: Boolean }];
```
Comment (snitin315) [inline packages/webpack-cli/lib/groups/runHelp.js:98]: I'm afraid this will conflict with #2151 😞

**Review by rishabh3112 — APPROVED** [review]

**Pull request merged**